src/components/CheckboxSvg/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export const checkboxSvgDataTestIds = {
+  base: 'checkbox-svg__base',
+};
+
+const CheckboxSvg = (): JSX.Element => {
+  return (
+    <svg
+      className="absolute pointer-events-none hidden peer-checked:block mt-1 outline-none"
```

Comment:
We can make the `className` to a parameter to customize the style later 🙏

Reply:
@hoangmirs thanks, updated it 🙏